weeklymission2/src/Components/FolderMain.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import searchIcon from "../images/search.svg";
+import { FolderCard } from "./FolderCard";
+import { FolderMenu } from "./FolderMenu";
+import { HandleFolder } from "./HandleFolder";
+import { useFiles } from "../useHooks/useFiles";
+import { useFolder } from "../useHooks/useFolder";
+
+export function FolderMain() {
+  const [menu, setMenu] = useState("전체");
+
+  const handleMenuChange = (newMenu, id) => {
+    setMenu(newMenu);
+    fetchFolder(id);
+  };
+
+  const folderNames = useFiles();
+  const { folder, fetchFolder } = useFolder();
```

Comment:
코드가 위에서 아래로 진행되기 때문에 handleMenuChange가 아래로 오는게 저는 좀 익숙하더라구요.
setMenu는 `handlemenuChange`위에 있고, `fetchFolder`는 아래에 있으니까 조금 혼동이 오네요 ㅎㅎ 

```suggestion
  const folderNames = useFiles();
  const { folder, fetchFolder } = useFolder();
  
  
    const handleMenuChange = (newMenu, id) => {
    setMenu(newMenu);
    fetchFolder(id);
  };

```